spec/requests/api/v2/providers/courses/publishable_spec.rb

```diff
@@ -43,23 +32,14 @@
     include_examples "Unauthenticated, unauthorised, or not accepted T&Cs"
 
     context "when course and provider is not related" do
-      let(:course) { create(:course) }
+      let(:course)   { create(:course, :unpublished_with_primary_maths) }
+      let(:provider) { create(:provider) }
 
       it { should have_http_status(:not_found) }
     end
 
-    context "unpublished course with draft enrichment" do\
-      let(:enrichment) { build(:course_enrichment, :initial_draft) }
-      let(:primary_with_mathematics) { find_or_create(:primary_subject, :primary_with_mathematics) }
-      let(:site_status) { build(:site_status, :new) }
-      let!(:course) do
-        create(:course,
-               provider: provider,
-               site_statuses: [site_status],
-               enrichments: [enrichment],
-               age: 17.days.ago,
-               subjects: [primary_with_mathematics])
-      end
+    context "unpublished course with draft enrichment" do
+      let(:course) { TestSetup.unpublished_course_with_draft_enrichment }
```

Comment:
Is the intention that we stop using `build`/`create` entirely? Or will still use build/create for interesting edge cases

Reply:
I think that that's a re-stating of the granularity question above, and ya ... we would still use create/build for interesting edge cases, aiming to just cover the majority of the types of courses in `TestSetup` but not all.